RELEASING.md

```diff
@@ -1,7 +1,7 @@
 1. Update to the latest SDK versions in `scripts/build.js`, `RNPurchases.podspec` and `android/build.gradle`.
 1. Update versions in VERSIONS.md.
 1. Update version in `package.json`.
-1. Update `platformFlavorVersion` in `RNPurchases.m` and `RNPurchasesModule.java`.
+1. Update `platformFlavorVersion` in `RNPurchases.m` and `PLUGIN_VERSION`in RNPurchasesModule.java`.
```

Comment:
Thanks for updating this!

Reply:
I think this step is missing tho. I don't see any updates to neither `RNPurchases.m` nor `RNPurchasesModule.java`.

Reply:
wowow. forgetting to press save in vscode has gotten me a few times now

Reply:
It happens. I activated the Autosave feature in the File menu because it also caused me trouble before